### ISSUE TITLE
Main file in 1.5.2 on npm appears broken

### ISSUE THREAD

**Issue by perliedman** [perliedman/leaflet-control-geocoder#146]
An error message like this is shown:

```
Cannot find module '../util' from 'C:\\Users\\per\\Documents\\source\\tellus-map-widget\\node_modules\\leaflet-control-geocoder\\dist'
```

**Comment by jcoupey** [comment]
Hi, first of all, thank you for taking the time to release 1.5.2!

I gave it a try and was about to report about something like this too.

On `npm install` I'm getting:

```
> leaflet-control-geocoder@1.5.2 postinstall [project_folder]/node_modules/leaflet-control-geocoder
> sh ./scripts/build.sh

Error: Cannot find module 'es3ify' from '[project_folder]/node_modules/leaflet-control-geocoder'
```

so the command at https://github.com/perliedman/leaflet-control-geocoder/blob/v1.5.2/scripts/build.sh#L5 fails and the file `dist/Control.Geocoder.js` is empty.

I can bypass this if I install `es3ify` as a dependency on my project level (then the same occurs for `browserify-shim`) but it doesn't feel quite right. And if I do so, I end up with an error similar to the one you report:

```
Error: Cannot find module './geocoders/nominatim' from '[project_folder]/node_modules/leaflet-control-geocoder/dist'
```

I'm sorry that my `npm` packaging skills don't allow me to help here, but I'm happy to test or investigate if required!

**Comment by perliedman** [comment]
Yeah, I was a bit too quick here. The goal was to remove dependency on `es3ify` and `browserify-shim` for users of the plugin, and only use them when building. Fixing that up on a friday afternoon apparently wasn't ideal :stuck_out_tongue_winking_eye: 

I'll try to find to look into fixing this...

**Comment by perliedman** [comment]
Ok, hopefully fixed this in 1.5.3, please try it out!

**Comment by jcoupey** [comment]
Works like a charm, thanks! \o/
